testsuite/features/support/env.rb

```diff
@@ -587,6 +587,18 @@ def process_code_coverage
   skip_this_scenario unless $is_containerized_server
 end
 
+# only test for excessive SCC accesses if SCC access is being logged
+Before('@srv_scc_access_logging') do
+  skip_this_scenario unless scc_access_logging_grain?
```

Comment:
I would have preferred a global variable rather than running this function each time